roles/nuage_master/tasks/certificates.yml

```diff
@@ -10,7 +10,7 @@
 
 - name: Create the req file
   command: >
-    openssl req -key "{{ nuage_ca_master_rest_server_key }}" -new -out "{{ nuage_mon_rest_server_crt_dir }}/restServer.req" -subj "/CN={{ ansible_nodename }}"
+    openssl req -key "{{ nuage_ca_master_rest_server_key }}" -new -out "{{ nuage_mon_rest_server_crt_dir }}/restServer.req" -subj "/CN={{ openshift_master_cluster_hostname }}"
```

Comment:
You could go that route OR in the case of HA use the `openshift.master.cluster_hostname` variable (assuming internal traffic) and fall back to using the first master's hostname (`groups.oo_first_master.0`) when not HA.

What do you think @detiber?